src/components/localization/locale-context.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import GlobalStyle from '../../themes/global-style'
```

Comment:
```suggestion
import GlobalStyle from 'themes/global-style'
```

using absolute path instead